.github/workflows/default.yml

```
@@ -19,7 +19,7 @@ jobs:
         with:
           node-version: 14
       - name: Install dependencies
-        run: npm ci
+        run: npm install && npm run build
```

`npm ci` (clean install) should have already run the `prepare` lifecycle script, so we don't need to run the build script again.

```suggestion
        run: npm ci
```